README.md

```diff
@@ -81,7 +81,7 @@ The topics will be inferred by the browser. The browser will leverage a classifi
     * The reason that each site gets associated with only one of the user's topics for that epoch is to ensure that callers on different sites for the same user see different topics. This makes it harder to reidentify the user across sites.
         * e.g., site A might see topic ‘cats’ for the user, but site B might see topic ‘automobiles’. It’s difficult for the two to determine that they’re looking at the same user.
     * The beginning of a week is per-user and per-site. That is, for the same user, site A may see the new week's topics introduced at a different time than site B. This is to make it harder to correlate the same user across sites via the time that they change topics.
-* Not every API caller will receive a topic. Only callers that observed the user visit a site about the topic in question within the past three weeks can receive the topic. If the caller (specifically the site of the calling context) did not call the API in the past for that user on a site about that topic, then the topic will not be included in the array returned by the API. The exception to this filtering is the 5% random topic, that topic will not be filtered.
+* Not every API caller will receive a topic. Only callers that observed the user visit a site about the topic in question within the past three weeks can receive the topic. If the topic in question was not observed by the caller, but one of its ancestor topics in the taxonomy hierarchy was observed by the caller, then the caller can also receive the closest ancestor topic that was observed by the caller. If the caller (specifically the site of the calling context) did not call the API in the past for that user on a site about that topic and its ancestor topics, then the topic will not be included in the array returned by the API. The exception to this filtering is the 5% random topic, that topic will not be filtered.
```

Comment:
Perhaps leave this paragraph alone and instead add a separate bullet as a note: Note that observing a topic also includes observing the topics entire ancestry tree. For instance, observing `/Arts & Entertainment/Humor/Live Comedy` also counts as having observed `/Arts & Entertainment/Humor/` and `/Arts & Entertainment`.

WDYT?

Reply:
Done.

Reply:
Aa